sdk/messaging/azeventhubs/README.md

```diff
@@ -0,0 +1,130 @@
+# Azure Event Hubs Client Module for Go
+
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a big data streaming platform and event ingestion service from Microsoft. For more information about Event Hubs see this link: [link](https://docs.microsoft.com/azure/event-hubs/event-hubs-about).
+
+Use the client library `github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs` in your application to:
+
+- Send messages to an event hub.
+- Consume messages from an event hub.
+
+**NOTE**: This library is currently a beta. There may be breaking interface changes until it reaches semantic version `v1.0.0`.
+
+Key links:
+- [Source code][source]
+- [API Reference Documentation][godoc]
+- [Product documentation](https://azure.microsoft.com/services/event-hubs/)
+- [Samples][godoc_examples]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Event Hubs client module for Go with `go get`:
+
+```bash
+go get github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
+```
+
+### Prerequisites
+
+- Go, version 1.18 or higher
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- A [Event Hub namespace](https://docs.microsoft.com/azure/event-hubs/).
+- An Event Hub. You can create an event hub in your Event Hubs Namespace using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create), or the [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli).
+
+### Authenticate the client
+
+The Event Hubs clients can be created using an Event Hubs connection string or a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
```

Comment:
We want them using a credential to connect over using connection strings.  I'd move the credential to be first and move connection strings to be second in the list or perhaps as an alternative note.

```suggestion
Event Hub clients are created using an Event Hub connection string or a credential from the [Azure Identity package][azure_identity_pkg], like [DefaultAzureCredential][default_azure_credential].
```

Reply:
Done. I basically just added the connection string on as a second sentence.